[doctest] Segmentation fault (core dumped) make builds fail randomly

Some PRs (https://travis-ci.org/github/qgis/QGIS-Documentation/pull_requests) are failing for the following reason (and this does not happen systematically)
```
Document: docs/user_manual/auth_system/auth_workflows
-----------------------------------------------------
Makefile:43: recipe for target 'doctest' failed
make: *** [doctest] Segmentation fault (core dumped)
docker.mk:4: recipe for target 'doctest' failed
make: *** [doctest] Error 2
The command "make -f docker.mk doctest" exited with 2.
cache.2
store build cache
```
Anyone knows what the issue could be?

--- a/docs/user_manual/auth_system/auth_workflows.rst
+++ b/docs/user_manual/auth_system/auth_workflows.rst
@@ -390,7 +390,7 @@ working for your server's connection, you can manually trigger a connection via
 the **Python Console** by running the following code (replace
 ``https://bugreports.qt-project.org`` with the URL of your server):
 
-.. testcode:: auth_workflows
+.. code-block:: python
 
    from qgis.PyQt.QtNetwork import QNetworkRequest
    from qgis.PyQt.QtCore import QUrl

